### ISSUE TITLE
Any alternate to sun database? How about Indian License plates?

### ISSUE THREAD

**Issue by sachinprabhu007** [matthewearl/deep-anpr#20]
Is there any alternative for sun database? 36gb is huge! I am interested to train for Indian License plates

**Comment by sourabh2k15** [comment]
I can provide you a 3gb zip containing the extracted backgrounds from SUN database. Also these are just backgrounds which are then utilized by gen.py to create test folder , license plates are randomly generated through code. I have modified the code to generate indian license plates ( 2 letters, 2 digits followed by 2 letters 4 digits ) . I am training the network now, if the loss converges I will upload the weights somewhere for everybody to use directly .

**Comment by sachinprabhu007** [comment]
Hey Sourabh! Thanks for the response. Yea I am trying to train system with license plates which I have captured. I was able to download sun database n check how it works. But was not successful in recognizing license plates

**Comment by sourabh2k15** [comment]
I am trying to do the same , but just with the difference that, I am not using captured images, I have just modified gen.py to put random number plates in indian formats onto backgrounds from SUN database . Before it was doing the same, but it was putting uk number plate format plates . This is our test folder which is used as training data for the network, lets see how it goes

**Comment by ab-si** [comment]
@sourabh2k15 

Could you please provide the zip containing the extracted backgrounds for SUN database, as 36GB is too heavy to download.

**Comment by sourabh2k15** [comment]
https://drive.google.com/file/d/0B_0Fk9Vc5r9ON0pnX3ctY1lKZEU/view?usp=sharing

**Comment by Abhi001vj** [comment]
can you share the weights.
